main.go

```diff
@@ -78,6 +78,10 @@ var (
 func main() {
 	var metricsLabels util.ArrayFlags
```

Comment:
We should move all flag variables into the `var` block above.

Reply:
Sure, will move it